src/routes.jsx

```diff
@@ -175,13 +177,22 @@ export default function makeRoutes(requireAuth = () => {}) {
         </Route>
       </Route>
       <Route path="login" component={Login} />
+      <Route
+        path="adminOrganizations"
```

Comment:
maybe just "organizations" so we don't need caps in the paths

Reply:
sure